sentry-spring/src/main/java/io/sentry/spring/tracing/SpringMvcTransactionNameProvider.java

```diff
@@ -25,4 +26,10 @@ public final class SpringMvcTransactionNameProvider implements TransactionNamePr
       return null;
     }
   }
+
+  @Override
+  @ApiStatus.Internal
+  public @NotNull TransactionNameSource provideTransactionSource() {
```

Comment:
Maybe we'd rename `TransactionNameProvider` then since now it provides `name` and `source`.

Reply:
This way it's not a breaking change since the new method has a default implementation.